### ISSUE TITLE
Zeri's W.Cast sometimes cast in wrong position

### ISSUE THREAD

**Issue by jacklee-code** [ensoulsharp-io/EnsoulSharp.Dependencies#43]
The method W.Cast(pred.PredictPosistion) / W.Cast(target.Position) both not work in Zeri. Her W Spell is always cast in the wrong position. I guess it may be the core problem?

**Comment by krisoft20** [comment]
> The method W.Cast(pred.PredictPosistion) / W.Cast(target.Position) both not work in Zeri. Her W Spell is always cast in the wrong position. I guess it may be the core problem?

it's not core problem, you need to pass start position and position for that spell:
`W.Cast(GameObjects.Player.Position, target.Position);`

also would be nice if you in start position do extend to target pos by few units

**Comment by jacklee-code** [comment]
> > The method W.Cast(pred.PredictPosistion) / W.Cast(target.Position) both not work in Zeri. Her W Spell is always cast in the wrong position. I guess it may be the core problem?
> 
> it's not core problem, you need to pass start position and position for that spell: `W.Cast(GameObjects.Player.Position, target.Position);`
> 
> also would be nice if you in start position do extend to target pos by few units

Oic, let me try again. Why do Zeri W need to pass the start position. The spell of other champion don't seem to need to pass the start position in method Cast()

**Comment by krisoft20** [comment]
well, seems zeri is special one, if you don't use like that it will be buggy so do it.

**Comment by EnsoulSharp** [comment]
Zeri's W same as Viktor's E on native so @krisoft20 is right.
